components/Layout.js

```diff
@@ -26,8 +26,9 @@ injectGlobal`
     margin: 0;
     padding: 0;
     font-family: "Fira Sans";
+    font-size: 14px;
     height: 100%;
-    background-color: ${theme.colors.white};
+    background-color: #ffffff;
```

Comment:
As mentioned via voice I think it's fine to not use constants for things like #fff (pure white) or #000 (pure black)